charts/agents-nuxt-service/values.yaml

```diff
@@ -0,0 +1,60 @@
+# Default values for agents-graphql-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from k8s-at-home common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+controller:
+  # -- Number of desired pods.
+  # We use 3 minimum to assure no outage durring rollout/preemtible node restarts
+  replicas: 3
+
+image:
+  # -- image repository
+  repository: "eu.gcr.io/logical-sled-220910/strg-at/agents-nuxt-service"
+  # -- image tag
+  tag: ""
+  # -- image pull policy
+  # pullPolicy: IfNotPresent
+
+securityContext:
+  capabilities:
+    drop:
+      # -- drop all privileges as we dont need them
+      - ALL
+  # -- set root fs to read only for security reasons
+  readOnlyRootFilesystem: true
+  # -- do not allow privilege escalation for security reasons
+  allowPrivilegeEscalation: false
+  # -- do not run as root for security reasons
+  runAsNonRoot: true
+  # -- run as user with <id>
+  runAsUser: 1000
+
+env:
+  # -- The timezone in the container
+  TZ: UTC
+  # -- The application port
+  PORT: "3000"
+  # -- The default node environment
+  NODE_ENV: "production"
+  # Url for connection to graphql
+  GRAPHQL_URL: "http://localhost:4000"
+  # Host for keycloak authentication
+  KEYCLOAK_HOST: "http://localhost:8180"
+  # Identifier for client
+  KEYCLOAK_CLIENT_ID: client-app
+  # Home url for login page
+  HOME_URI: http://localhost:3000/login
```

Comment:
mh isn't this always related to keycloak_host? do we want to set this explicit here?

Reply:
@tyriis  should i remove or keep variable commented for now?

Reply:
yeah comment and variable should stay, contetn not

Reply:
@tyriis  should I make empty also KEYCLOAK_CLIENT_ID ?

Reply:
yeah would make sense with a check